includes/class-sensei-course.php

```diff
@@ -503,6 +503,11 @@ public function course_video_meta_box_content() {
 	public function meta_box_save( $post_id ) {
 		global $post;
 
+		/* Check nonce exsit before verify. */
+		if ( ! isset( $_POST[ esc_attr( 'woo_' . $this->token . '_noonce' ) ] ) ) {
```

Comment:
`esc_attr` isn't needed here.

Also, you could put `$_POST[ 'woo_' . $this->token . '_noonce' ]` in a variable since it's used more than once and replace every instance with the variable instead.

Reply:
Ok, I do all this

Reply:
Ah right. No need for the variable in that case.